pages/donate.md

```diff
@@ -1,80 +1,99 @@
+Qubes is one of the few operating systems that places the security of its users above all else.
+It has the potential to be a worldwide game changer in endpoint security.
+But for that to happen, the project requires a wide spectrum of expertise.
+In order to both retain and expand our core team, we require substantial funding from supporters like you.
 
-You can donate towards Qubes development using your credit card through
-[Open Collective] (which uses [Stripe][stripe] to process financial
-transactions). Please note that, unlike [Bitcoin donations], we lose
-[~14%][open-collective-faq] of every credit card donation to administrative
-overhead.
+With the help of our community, we hope to eventually build a nonprofit organization that will ensure the long-term future of the Qubes Project.
 
----
+### How is My Donation Used?
+Your donation pays directly for individual developers who have been hired to work on the open source edition of Qubes.
+
+We regret that we cannot implement requested features based on funding (if everybody were to decide, then nothing would get implemented).
+
+### Is My Donation Tax Deductible?
+Donations to the Qubes project are not tax deductible.
+
+### Can I Donate Something Other Than Money?
+We prefer monetary contributions which allow us greater flexibility.
+However, if you or your organization has relevant knowledge, experience, leads, or resources and would like to help us secure the future of Qubes OS, don’t hesitate to [contact us][contact].
+
+### <a name="verify-address"></a>How Do I Verify Your Bitcoin Address?
+The donation address can be verified via the [Qubes Security Pack][Qubes Security Pack], specifically in the [fund directory][fund].
+We provide [detailed instructions for verifying the digital signatures][verify].
+You can also view the address on [blockexplorer.com][blockexplorer.com] and [blockchain.info][blockchain.info].
 
-## About Your Donation
+The Qubes Project maintains a [decentralized Bitcoin fund using a multi-signature wallet][announcement].
+This means that no single person is capable of spending these funds.
 
-- We will use your donations to fund our continued development of Qubes.
-- We regret that we cannot implement requested features based on funding. (If
-  everybody were to decide, then nothing would get implemented.)
-- Donations to the Qubes project are not tax-deductible.
+### What is Qubes?
+Qubes OS is a free and open source operating system that focuses on security and privacy.
+Its architecture is built to enable you to define and visualize different security environments on your computer and control how they interact, helping you to get your work done safely.
+
+Visit our [introduction page to Qubes][intro page] or watch a [video overview of the system][video page] to learn more.
+
+[Bitcoin donations]: #bitcoin
+[stripe]: https://stripe.com
 
-[funding]: /funding/
+[open-collective-faq]: https://opencollective.com/faq
+[verify address]: #
```

Comment:
Line 87? It should be the linked to the anchor on that page: 

> ### <a name="verify-address"></a>How Do I Verify Your Bitcoin Address?

Thanks for catching that!